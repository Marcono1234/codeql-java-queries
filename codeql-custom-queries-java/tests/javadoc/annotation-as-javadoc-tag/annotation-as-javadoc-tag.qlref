./javadoc/annotation-as-javadoc-tag.ql
