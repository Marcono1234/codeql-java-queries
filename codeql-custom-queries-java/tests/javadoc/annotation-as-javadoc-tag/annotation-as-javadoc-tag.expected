| Test.java:2:4:2:15 | @Deprecated | Annotation being mistaken for javadoc tag |
| Test.java:8:8:8:17 | @Override | Annotation being mistaken for javadoc tag |
